### PR TITLE
Feat: UI to answer the questions in Q&A Section Added

### PR DESCRIPTION
**Pull request by BHS-Harish**
## Issue that this pull request solves

 Closes: #886 

## Screenshots

https://github.com/user-attachments/assets/078b6e32-3943-4e49-8836-214d7984ced0

### CODE REVIEW

File: frontend/src/service/Faq.jsx [hunk 1]
```diff
@@ -326,3 +326,26 @@ export const downvote = async (questionId, handleToast) => {
...
+    const response = await fetch(url, {
+      method: "POST",
+      headers: {
+        "content-type": "application/json",
+      },
+      body: JSON.stringify(data),
+    });
+    const res = await response.json();
+    if(response.status==200)
+      showToast(setToast, "Answer posted successfully","success");
```
Comment (Kajol-Kumari) [inline frontend/src/service/Faq.jsx:343]: Can you show the message like “Thanks for answering, it has been sent to admins for review and will appear here on approval “

Comment (BHS-Harish) [reply frontend/src/service/Faq.jsx:343]: Okay i will change it....But Build not shown any error in local machine.. it failing here

**Comment by Kajol-Kumari** [comment]
Also, please check the deploy build failure as well.

**Comment by BHS-Harish** [comment]
> Also, please check the deploy build failure as well.

No error shown while building on local....

**Pull request closed without merging**